src/Stratis.Bitcoin.IntegrationTests.Common/Runners/BitcoinCoreRunner.cs

```diff
@@ -53,6 +54,15 @@ public override void Start()
             string logMode = Debugger.IsAttached ? "-debug=net" : string.Empty;
             TimeSpan duration = TimeSpan.FromSeconds(15);
 
+            // The complete path bitcoind uses to locate (e.g.) the block files consists of the source code build folder,
```

Comment:
e.g in brackets ? Not too sure what that means.

Reply:
You can read it as "The complete path bitcoind uses to locate, for example, the block files,".